target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts

```diff
@@ -216,6 +216,11 @@
 			label = "lan1";
 		};
 
+		port@1 {
+			reg = <1>;
+			label = "eth1";
```

Comment:
If port 1 on the switch is eth1 now, is `gmac1` still used? If it is not used you should remove it from the device tree.